### PR TITLE
log status changes

### PR DESCRIPTION
**Pull request by denniswittich**

### CODE REVIEW

File: learning_loop_node/detector/detector_node.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def __init__(self, name: str, detector: DetectorLogic, uuid: Optional[str] = Non
         self.log.info('Using %s/%s', self.organization, self.project)
         self.operation_mode: OperationMode = OperationMode.Startup
         self.connected_clients: List[str] = []
+        self.previous_state: Optional[str] = None
```
Comment (NiklasNeugebauer) [inline learning_loop_node/detector/detector_node.py:47]:
```suggestion
```

File: learning_loop_node/trainer/trainer_node.py [hunk 2]
```diff
@@ -21,6 +21,7 @@ def __init__(self, name: str, trainer_logic: TrainerLogicGeneric, uuid: Optional
         self.trainer_logic = trainer_logic
         self.last_training_io = LastTrainingIO(self.uuid)
         self.trainer_logic._last_training_io = self.last_training_io
+        self.previous_state: Optional[str] = None
```
Comment (NiklasNeugebauer) [inline learning_loop_node/trainer/trainer_node.py:24]:
```suggestion
```

**Review by NiklasNeugebauer — CHANGES_REQUESTED** [review]
I think these are not needed since they are defined in the parent class.

**Review by NiklasNeugebauer — COMMENTED** [review]
Looks good otherwise

File: learning_loop_node/detector/detector_node.py (hunk 1 above)
Comment (denniswittich) [reply learning_loop_node/detector/detector_node.py:47]: True.. didn't clean up  after moving these to parent

**Review by NiklasNeugebauer — APPROVED** [review]

**Pull request merged**